src/IConfigOptions.ts

```diff
@@ -50,6 +50,8 @@ export interface IConfigOptions {
         welcome_background_url?: string | string[]; // chosen at random if array
         auth_header_logo_url?: string;
         auth_footer_links?: { text: string; url: string }[];
+        title_template?: string;
```

Comment:
Branding seems to consist of mostly urls, but it felt like a reasonable place to put this. Shout if this should be somewhere else.